lib/modules/apostrophe-ui/public/css/global/generic.less

```diff
@@ -0,0 +1,9 @@
+// ===============================================================
+// Generic Styles for Apostrophe UI
+// ===============================================================
+
+* {
```

Comment:
Is this imported inside something that will scope it appropriately?

Is @stuartromanek among the reviewers?